### PR TITLE
Add Channel "Let's Talk About Stuff"

### PR DESCRIPTION
**Pull request by Yesfinethankyou**
Added some more channels so that we could work towards getting the Best of Bread Tube 2018 playlist set up.

https://www.reddit.com/r/BreadTube/comments/ai8d6x/best_of_breadtube_2018_as_voted_by_this_sub/

### CODE REVIEW

**Comment by j16r** [comment]
@dirkkelly might have added most of these in #33

File: data/channels.yml [hunk 1]
```diff
@@ -262,3 +262,40 @@
...
+  tags: ["breadtube"]
+-
+  name: "Lindsay Ellis"
+  slug: "lindsayellis"
+  url: "https://www.youtube.com/channel/UCG1h-Wqjtwz7uUANw6gazRw"
+  subscribers: 548886
+  tags: []
+ -
+  name: "Richard Wolff"
+  slug: "RichardDWolff"
```
Comment (craiglonsdale) [inline data/channels.yml]: Should be lower case for sake of consistency.

File: data/channels.yml [hunk 2]
```diff
@@ -262,3 +262,40 @@
...
+  url: "https://www.youtube.com/channel/UCSkzHxIcfoEr69MWBdo0ppg"
+  subscribers: 66067
+  tags: ["breadtube"]
+-
+  name: "Lindsay Ellis"
+  slug: "lindsayellis"
+  url: "https://www.youtube.com/channel/UCG1h-Wqjtwz7uUANw6gazRw"
+  subscribers: 548886
+  tags: []
+ -
```
Comment (craiglonsdale) [inline data/channels.yml]: The `-` tag needs to be moved over as this is causing compiation issues

**Comment by dirkkelly** [comment]
Thank you so much for your submission @Yesfinethankyou, great first start with adding some excellent channels.

As @craiglonsdale mentioned there are a few syntax issues which comes with the territory of dealing with programming. Regardless of the specifics though you really did all the things necessary to get your first BreadTube.tv pull request together and we really appreciate it!

For a reference to all the things mentioned above please check out https://github.com/breadtubetv/breadtubetv/pull/33/files which is the Pull Request in which I added every channel (hopefully I got them all) from the [r/breadtube](https://www.reddit.com/r/BreadTube/comments/9kwkxa/master_list_of_leftwing_youtube_and_podcast/) master list.

The one missing from this list is `let's talk about stuff.`

Could you please update this pull request to 

- Remove all other channels (now that they're already in)
- Merge in the recent master changes
- Add the video image

Let me know if you'd like my help with this, or for me to just do this for you. There's also a video on my channel guiding you through the whole process. https://www.youtube.com/watch?v=jpOun7YXFpg

Thanks!

**Comment by dirkkelly** [comment]
Closing for #65

**Pull request closed without merging**